controllers/course.js

```diff
@@ -2,9 +2,9 @@ const courseService = require('~/services/course')
 
 const getCourses = async (req, res) => {
   const { id: author } = req.user
-  const { skip, limit } = req.query
+  const { skip, limit, sort } = req.query
 
-  const course = await courseService.getCourses({ skip: parseInt(skip), limit: parseInt(limit), author })
+  const course = await courseService.getCourses({ skip: parseInt(skip), limit: parseInt(limit), author, sort })
```

Comment:
keep in mind that for now getCourses accepts arguments without destructuring (match, skip, limit). 
What do you think maybe try use getSortOptions fn for sort and pass sortOptions instead ?